Sources/OpenAIAPIManager/OpenAIAPIManagerEndpoints.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 // MARK: - API Enum
 /// Enum defining the base URL for an API.
-enum ChatGPTAPIEndpoint {
+enum OpenAIAPIManagerEndpoints {
```

Comment:
The new name of endpoints class should be OpenAIAPIEndpoints instead of OpenAIAPIManagerEndpoints.